src/frontend/screens/Settings/components/SettingsModal/index.scss

```diff
@@ -10,3 +10,17 @@
     width: 100%;
   }
 }
+
```

Comment:
the issue was caused by some CSS to make the settings dialog use more height in small screens, I removed some css in the other file and implemented that here we some media queries.